Makefile.core.mk

```diff
@@ -375,7 +375,9 @@ copy-templates:
 	cp manifests/charts/istio-control/istio-discovery/templates/istiod-injector-configmap.yaml manifests/charts/istiod-remote/templates
 	cp manifests/charts/istio-control/istio-discovery/templates/configmap.yaml manifests/charts/istiod-remote/templates
 	cp manifests/charts/istio-control/istio-discovery/templates/_helpers.tpl manifests/charts/istiod-remote/templates
-	sed -e '1 i {{- if .Values.global.configCluster }}' -e '$$ a {{- end }}' manifests/charts/base/crds/crd-all.gen.yaml > manifests/charts/istiod-remote/templates/crd-all.gen.yaml
+	sed -e '1 i {{- if and .Values.global.configCluster (eq .Values.base.releaseChannel "stable") }}' -e '$$ a {{- end }}' manifests/charts/base/files/stable.gen.yaml > manifests/charts/istiod-remote/files/stable.gen.yaml
+	sed -e '1 i {{- if and .Values.global.configCluster (or (not .Values.base.ReleaseChannel) (eq .Values.base.ReleaseChannel "extended")) }}' -e '$$ a {{- end }}' manifests/charts/base/files/extended.gen.yaml > manifests/charts/istiod-remote/files/extended.gen.yaml
```

Comment:
Why is there an "extended" and "crd-all"? In the current doc, there are two modes:
* What we have today, now named "extended"
* Stable

But here we have 3?

Reply:
Hmm seems in the generated code we do have 2. Do we need L380?

Reply:
`crd-all` should be extended. There's only two modes. This will be fixed, there was a weird git diff that I was debugging by keeping the `crd-all` name for now.